phpBB/phpbb/db/migration/data/v400/rename_duplicated_index_names.php

```diff
@@ -27,7 +27,7 @@ class rename_duplicated_index_names extends migration
 	/**
 	 * @var array
 	 */
-	protected static $rename_index;
+	protected static $rename_index = [];
```

Comment:
This way `!isset(self::$rename_index)` below will always return false and rename index array will never be filled with actual data.
Probably it would be better adding `self::$rename_index = [];` right after `if (!isset(self::$rename_index)) {`, around line 43.